Makefile.dep

```diff
@@ -457,7 +466,9 @@ endif
 
 ifneq (,$(filter stdin,$(USEMODULE)))
   ifneq (,$(filter stdio_uart,$(USEMODULE)))
-    USEMODULE += stdio_uart_rx
+    ifeq (,$(filter stdio_nimble,$(USEMODULE)))
```

Comment:
how about `ifneq (,$(filter stdio_uart stdio_nimble,$(USEMODULE)))`?

Reply:
I'm reverting to pre-PR state, because UART-in-parallel feature is removed.